tests/CSharpTemp/CSharpTemp.h

```diff
@@ -509,3 +509,27 @@ class DLL_API TestOverrideFromSecondaryBase : public Foo, public SecondaryBase
     void VirtualMember();
     void setProperty(int value);
 };
+
+class DLL_API TestParamToInterfacePassBaseOne
+{
+};
+
+class DLL_API TestParamToInterfacePassBaseTwo
+{
+	int m;
```

Comment:
You don't initialise this when using the empty ctor. For some reason it happens to work but we cannot rely on that. Please assign it to 0 in the empty ctor.